client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -811,6 +812,7 @@ const mapStateToProps = (
 		...pressThisData,
 		answer_prompt: getQueryArg( window.location.href, 'answer_prompt' ),
 		completedFlow,
+		...( canvasEditMode && { canvas: canvasEditMode } ),
```

Comment:
I think we should simply assign this at all times. We don't want to be sending people into the template selection mode at all, ever.

Reply:
This seems sensible to me. But one concern is that changes goes well beyond our current use case and seems to change the calypso site editor handling/display for all cases. That seems like a big change, and I I wonder if that warrants wider discussion beyond our team?

Reply:
I'm in agreement with Matt here. We don't want to surface this to our users by default in any entry the way it is right now. But we may be able to in the future after core has iterated on its development.

With that it's important to leave a clear comment that we are doing this.

Reply:
> This seems sensible to me. But one concern is that changes goes well beyond our current use case and seems to change the calypso site editor handling/display for all cases. That seems like a big change, and I I wonder if that warrants wider discussion beyond our team?

To me the big change was in Gutenberg that started exposing this to our users by default. It is still being iterated on, but is currently not in a place that is good for us to expose by default to our users on dotcom.

We relayed some of this feedback to core in this p2: p9Jlb4-6ad-p2#comment-6657, expressed desire to be able to bypass this, and were soon after notified that a change had been implemented that would allow us to do so.

We also reached out to team-calypso in slack to make them aware of this change, and present them the option to handle it on their end since they generally handle things around the gutenberg upgrade cycles. They responded that they would be happy for us to handle this, which we are happy to do 😁  p1673663392570019/1673546040.780839-slack-C7YPUHBB2